packages/file-asset-apis/test/s3/s3-helpers-spec.ts

```diff
@@ -129,7 +129,11 @@ describe('S3 Helpers', () => {
         // added s3-large-spec.ts to show how to use listS3Objects w/ continuation token
 
         it('should list delete an object', async () => {
-            const deleted = await s3Helpers.deleteS3Object(client, { Bucket: bucketName, Key: 'bad' });
+            const deleted = await s3Helpers.s3RequestWithRetry({
```

Comment:
I don't think you should be changing these tests like this.  If we have code calling `s3Helpers.deleteS3Object` like in the old code, we'd want to keep a test to make sure it works.  We shouldn't be changing the existing interface, merely extending it.  So keep the old tests and add new tests that call the `s3RequestWithRetry` if that seems appropriate.